packages/algob/src/internal/core/errors-list.ts

```diff
@@ -29,7 +29,8 @@ export const ERROR_RANGES = {
   BUILTIN_TASKS: { min: 600, max: 699, title: "Built-in tasks errors" },
   PLUGINS: { min: 800, max: 899, title: "Plugin system errors" },
 
-  SCRIPT: { min: 900, max: 999, title: "Script related errors" }
+  SCRIPT: { min: 900, max: 999, title: "Script related errors" },
+  PYTHON: { min: 1000, max: 1099, title: "Python (PyTeal) related errors" }
```

Comment:
```suggestion
  PyTEAL: { min: 450, max: 460, title: "Python (PyTeal) related errors" }
```

And you will have to update the numbers in the new errors accordingly.

Reply:
ok